src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Services/CohortDomainServiceTests.cs

```diff
@@ -312,6 +320,7 @@ public CohortDomainServiceTestFixture()
                 Db = new ProviderCommitmentsDbContext(new DbContextOptionsBuilder<ProviderCommitmentsDbContext>()
                     .UseInMemoryDatabase(Guid.NewGuid().ToString())
                     .ConfigureWarnings(warnings => warnings.Throw(RelationalEventId.QueryClientEvaluationWarning))
+                    .EnableSensitiveDataLogging()
                     .Options);
```

Comment:
Not seen that function before. How do I use it?

Reply:
Enables application data to be included in exception messages, logging, etc. This can include the values assigned to properties of your entity instances, parameter values for commands being sent to the database, and other such data.